docker/Dockerfile

```diff
@@ -6,13 +6,32 @@ RUN apk add --update git && \
     apk add curl && \
     apk add bash && \
     apk add openjdk8 && \
-    apk add openjdk8-doc
+    apk add openjdk8-doc && \
+    apk add expect
 
 # Set sbt and java env vars
 ENV SBT_VERSION 0.13.11
 ENV SBT_HOME /usr/local/sbt
 ENV PATH ${PATH}:${SBT_HOME}/bin
 ENV PATH ${PATH}:/usr/lib/jvm/java-1.8-openjdk/bin
+ENV SCALA_VERSION=2.11.8
+ENV SCALA_HOME=/usr/share/scala
```

Comment:
The scala script does automatic caching by creating jars, thus is assumed to be faster and more more friendly.

> we're polluting something with outside dependencies that can be resolved through PathResolver.scala magic

I don't get the point. I have scala install on my machine, it doesn't seem to pollute anything?